tbx/project_styleguide/templates/patterns/pages/work/work_page.html

```diff
@@ -10,14 +10,26 @@
 {% endblock %}
 
 {% block main_image %}
+    {% if page.remove_desaturation_on_header_image %}
+        {% image page.header_image fill-1600x700 format-webp as desktop_image %}
+        {% image page.header_image fill-1280x560 format-webp as desktop_image_retina %}
+        {% image page.header_image fill-400x225 format-webp as mobile_image %}
+        {% image page.header_image fill-800x450 format-webp as mobile_image_retina %}
```

Comment:
In retrospect I've decided to leave these as they are. It is the case we aren't serving the maximum image size we could for mobile, but otherwise the sizes get too large and we also need to be sustainable.